starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,56 @@
 // Names and Input
 
+let hacker1 = "Jonathan";
+console.log(`the driver's name is ${hacker1}`);
+
+let hacker2 = "Josh";
+console.log(`the navigator's name is ${hacker2}`);
 
 //Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `driver has the longest name, it has ${hacker1.length} characters`
+  );
+} else if (hacker1.length === hacker2.length) {
+  console.log(
+    `wow, you both got equally long names, your names have ${
+      hacker2.length
+    } characters`
+  );
+} else {
+  console.log(
+    `yo, navigator has the longest name, it has ${hacker2.length} characters`
+  );
+}
 
 // Lorem ipsum generator
+console.log("Driver's name");
+for (let i = 0; i < hacker1.length; i++) {
+  console.log(hacker1[i].toUpperCase());
+}
+
+console.log("Nav's name");
+for (let i = hacker2.length - 1; i >= 0; i--) {
+  console.log(hacker2[i]);
```

Comment:
This is printing all the letters in the backward order, but also you need to put it back to the string.